docs/releases/5.2.2.md

```diff
@@ -19,4 +19,8 @@ depth: 1
  * Ensure that Snippets search results correctly use the `index_results.html` or `index_results_template_name` override on initial load (Stefan Hammer)
  * Avoid error when attempting to moderate a page drafted by a now deleted user (Dan Braghis)
 
+### Other features
+
+ * Added formal Django 5.0 support (Dan Braghis)
```

Comment:
This should have its own section, e.g. https://github.com/wagtail/wagtail/commit/18e57574cbba60c6b26cc2523f38eb481dd9c15a#diff-fd9868873f77fa2da6a9d5a063eb95954dfeb17305c237c0cf61a676b6da4b95

Reply:
done